README.md

```diff
@@ -48,3 +48,10 @@ We use Tailwind for styling. As a rule of thumb we use use **component scoped st
 - add a reusable Tailwind component directive in `globals.css`
 
 _Avoid modifying the @base layer in `globals.css` (such as `span`, `p`, etc.) unless absolutely necessary. If doing so, make sure to smoke test element in app thoroughly to catch breaking changes._
+
+## End-to-end testing
+
+We use [GuardianUI](https://github.com/GuardianUI/GuardianTest) on top of [Playwright](https://playwright.dev/) for end-to-end testing. To set up for the first
+time, run `npx playwright install` after `npm install`. Then, to run the tests, run `npm run test:e2e` (or `npm run test:e2e:headless` for headless mode). Setting the `E2E_ENV` environment variable to production or test will run the tests against the production or test environment, respectively. By default, the tests will run against the local development environment. If you're developing on VSCode, you can install the [Playwright extension](https://playwright.dev/docs/getting-started-vscode) for ease of use.
```

Comment:
change `npm run test:gui` to `npm run test:e2e` in `package.json`?

Reply:
Yep I'll change to `npm run test:e2e`. The tests failed as the text "Thanks for being part of the Glo movement" on the login modal was removed.